pkgs/data/fonts/powerline-fonts/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation {
+  name = "powerline-fonts-39c99c";
+
+  src = fetchgit {
+    url = https://github.com/powerline/fonts.git;
+    rev = "39c99c02652f25290b64e24a7e9a7cfb8ce89a3e";
+    sha256 = "9c83a30f36dc980582c0a079bd2896f95d19e1cb0ba5afbd8cae936c944256dd";
+  };
+
+  buildPhase = "true";
+
+  installPhase =
+    ''
+      mkdir -p $out/share/fonts/opentype
+      cp -v */*.otf $out/share/fonts/opentype
+
+      mkdir -p $out/share/fonts/truetype
+      cp -v */*.ttf $out/share/fonts/truetype
+    '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/powerline/fonts;
+    description = "Patched fonts for Powerline users.";
+    longDescription = ''
+      Pre-patched and adjusted fonts for usage with the Powerline plugin.
+    '';
+    license = with licenses; [ asl20 free ofl ];
```

Comment:
Which part?  The multiple licenses thing?  Yes, apparently so (I found one or two other packages that did the same thing...), although it came up in PR #7493 as well and so I want to make a change here before we close this PR.  Any thoughts on how to handle these situations?